### PR TITLE
Add basic PWA setup

### PR DESCRIPTION
**Pull request by avee87**
Very basic PWA setup to make iOS bookmarks app-like:
IMG_24A33C686F35-1
Note the lack of browser navigation

Related to #25

### CODE REVIEW

File: front/nuxt.config.ts [hunk 1]
```diff
@@ -32,24 +31,24 @@ export default defineNuxtConfig({
     components: [
         {
             path: '~/components',
-            pathPrefix: false,
+            pathPrefix: false
```
Comment (avee87) [inline front/nuxt.config.ts:34]: This project really needs some Prettier configs. Formatting is all over the place and it was tricky to minimize changes from just saving files

File: front/nuxt.config.ts [hunk 2]
```diff
@@ -59,86 +58,46 @@ export default defineNuxtConfig({
...
-    //         type: 'module',
-    //     },
-    // },
+    pwa: {
+        registerType: 'autoUpdate',
+        manifest: {
+            name: 'Pico',
+            short_name: 'Pico',
+            theme_color: '#1867c0',
+            background_color: '#1867c0',
```
Comment (avee87) [inline front/nuxt.config.ts:81]: I took these colors from commented-out config. Would appreciate suggestions for more appropriate ones.

File: front/nuxt.config.ts (hunk 1 above)
Comment (cioraneanu) [reply front/nuxt.config.ts:34]: Added basic prettier config to main branch.

File: front/nuxt.config.ts (hunk 2 above)
Comment (cioraneanu) [reply front/nuxt.config.ts:81]: You can set them to white. Will need to make them dyanimic at runtime when dark theme is finished.

**Pull request merged**